### ISSUE TITLE
topk in multi_shallow_embedding

### ISSUE THREAD

**Issue by cherishleon** [liuxz1011/TodyNet#7]
in code layer.py class multi_shallow_embedding
The topk algorithm doesn't seem to work.
`

class GNNStack(nn.Module):
    def __init__(self, gnn_model_type, num_layers, groups, pool_ratio, kern_size, 
                 in_dim, hidden_dim, out_dim, 
                 seq_len, num_nodes, num_classes, dropout=0.5, activation=nn.ReLU()):

        super().__init__()
        
        # TODO: Sparsity Analysis
        k_neighs = self.num_nodes = num_nodes
        
        self.num_graphs = groups
        
        self.num_feats = seq_len
        if seq_len % groups:
            self.num_feats += ( groups - seq_len % groups )
        self.g_constr = multi_shallow_embedding(num_nodes, k_neighs, self.num_graphs)
`
k_neighs = num_nodes, 
So the example you gave doesn't take topk, and you also don't seem to notice the specific value of k in the paper
